tools/build_travis.py

```diff
@@ -140,6 +140,40 @@
      }
      ]
 
+linking_list = [
+    {"target": "K64F",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "eth"  : ["NET_1", "NET_2", "NET_3", "NET_4"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
+               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
+               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
+               }
+     }
+     ]
+
+linking_list = [
+    {"target": "K22F",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
+               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
+               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
+               }
+     }
+     ]
+
+linking_list = [
+    {"target": "KL43Z",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
+               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
+               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
+               }
+     }
+     ]
```

Comment:
@mmahadevan108 These all need to be in the same array. Please replace these changes with the following:

```
linking_list = [
    {"target": "LPC1768",
     "toolchains": "GCC_ARM",
     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
               "eth"  : ["NET_1", "NET_2", "NET_3", "NET_4"],
               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
               }
     },
    {"target": "K64F",
     "toolchains": "GCC_ARM",
     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
               "eth"  : ["NET_1", "NET_2", "NET_3", "NET_4"],
               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
               }
     },
    {"target": "K22F",
     "toolchains": "GCC_ARM",
     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
               }
     },
    {"target": "KL43Z",
     "toolchains": "GCC_ARM",
     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
               "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
               "usb"  : ["USB_1", "USB_2" ,"USB_3"],
               }
     }
     ]
```